contracts/modules/DecentHatsCreationModule.sol

```diff
@@ -202,13 +199,14 @@ contract DecentHatsCreationModule is DecentHatsUtils {
         );
 
         // Deploy Decent Autonomous Admin Module, which will wear the Admin Hat
-        address autonomousAdminModule = moduleProxyFactory.deployModule(
-            decentAutonomousAdminMasterCopy,
+        address autonomousAdmin = moduleProxyFactory.deployModule(
+            decentAutonomousAdminImplementation,
             abi.encodeWithSignature("setUp(bytes)", bytes("")),
             uint256(
                 keccak256(
                     abi.encodePacked(
-                        // for the salt, we'll concatenate our static salt with the id of the Admin Hat
+                        // for the salt, we'll concatenate our static salt
+                        // with the Admin Hat ID
                         SALT,
```

Comment:
This is not `SALT` though, it's `PEPPER` 😿

Reply:
WONTFIX